SHARE-249 Refactoring - Adminarea - APK Generation | Pending

hebesempso opened the pull request:
- Reset and reset all buttons should work now as intended
- Changed displayed int of apk status to what the status is written out
- List is now sorted after apk request time, which also gets reset after clicking the reset and reset all buttons
- Added filters
- Wrote tests

---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [x] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no warnings and errors 
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Verify that all tests are passing, if not please state the test cases in the [section](#Tests) below
- [x] Perform a self-review of the changes
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [x] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### Additional Description
`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information
`TODO: add additional information about testruns here`

hebesempso committed:
SHARE-249 Refactoring - Adminarea - APK Generation | Pending

- Reset and reset all buttons should work now as intended
- Changed displayed int of apk status to what the status is written out
- List is now sorted after apk request time, which also gets reset after clicking the reset and reset all buttons
- Added filters
- Wrote tests

dmetzner commented:
We enter the correct Jenkins configs on the auto-deploy server to fully tests this feature.

leno12 reviewed (changes requested):
The rebuild button is not working for me. The website crashes with the following error
The "App\Catrobat\Services\Ci\JenkinsDispatcher" service or alias has been removed or inlined when the container was compiled. You should either make it public, or stop using the container directly and use dependency injection instead.

dmetzner commented:
The feature should now work on our test server. 

![image](https://user-images.githubusercontent.com/40868718/83996182-c0cd0080-a95b-11ea-86fb-42a99165289f.png)

just rebase and force push again

hebesempso committed:
SHARE-249 Refactoring - Adminarea - APK Generation | Pending

- Reset and reset all buttons should work now as intended
- Changed displayed int of apk status to what the status is written out
- List is now sorted after apk request time, which also gets reset after clicking the reset and reset all buttons
- Added filters
- Wrote tests
